src/playstore-reviews.ts

```diff
@@ -0,0 +1,160 @@
+import { Data, PlayStoreConfig, PublishedReviews } from "./global-types"
+
+import { google, androidpublisher_v3 } from "googleapis"
+import { app, AppInformation } from "google-play-scraper"
+import { get } from "android-versions"
+
+export default class PlayStoreReviews {
+
+    SCOPES = ['https://www.googleapis.com/auth/androidpublisher']
+    STORE_NAME = "Play Store"
+
+    async fetch(config: PlayStoreConfig, publishedReviews: PublishedReviews): Promise<Data> {
+        let { id } = config
+
+        const appInformation = await app({ appId: id })
+        const result = await this.fetchPlayStoreReviews(id, config.publisherKey)
+
+        let reviews: Review[] = []
+        var newReviews = reviews.concat(...result).filter((review) => {
+            if (publishedReviews[id] !== undefined && publishedReviews[id].indexOf(review.id) >= 0) {
+                return false
+            }
+            return true
```

Comment:
Same as above. Returning boolean in if conditions is generally discouraged.

Reply:
Fixed.